iroh/src/magicsock/relay_actor.rs

```diff
@@ -605,7 +618,7 @@ impl ActiveRelayActor {
                     };
                     match msg {
                         Ok(msg) => {
-                            self.handle_relay_msg(msg, &mut state);
+                            self.handle_relay_msg(msg, &mut state).await;
```

Comment:
We can not make this async.  Actors can not do async stuff in their main loop.  We violate this in plenty of places, but this actor has finally been cleaned up.  We should not regress this.

Reply:
Right. But how would you apply backpressure then?

Reply:
Removed that commit again.

Reply:
> Right. But how would you apply backpressure then?

For completeness answering that here as well (it is mentioned above): by keeping the blocked message around in the actor state *somehow* and not polling for new messages from the relay while you have this pending message.

Of course in this case it means you're backpressuring the entire relay TCP stream while you only really want to backpressure disco messages.  But to really do something about this you'd have to make two tcp streams to the relay and... lots of things we don't want to do.